### PR TITLE
Readme for device plugins helm charts

### PR DESCRIPTION
**Pull request by obaranov1**
What parameters should I define for operator helm chart? In the values.yaml of operator there is only a reference to its image.

### CODE REVIEW

File: charts/qat-device-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Intel QAT Device Plugin Helm Chart
+
+## Get Helm Repository Info
+```
+helm repo add intel https://intel.github.io/helm-charts/
+helm repo update
+```
+You can then run helm search repo intel to see pulled charts.
```
Comment (poussa) [inline charts/qat-device-plugin/README.md:8]: put the `helm search repo intel` into single quotes so it looks like a command.  Same for all the charts.

File: charts/device-plugin-operator/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Intel Device Plugins Operator Helm Chart
+
+## Get Helm Repository Info
+```
+helm repo add intel https://intel.github.io/helm-charts/
+helm repo update
+```
+
+## Install Helm Chart
+In order to install device plugin Operator, its CRDs should be installed.
```
Comment (poussa) [inline charts/device-plugin-operator/README.md:10]: You should say that the CRDs *are* installed as part of the chart.

File: charts/device-plugin-operator/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+You can then run helm search repo intel to see pulled charts.
+
+## Upgrade Chart
+```
+helm upgrade device-plugin-operator intel/device-plugin-operator [flags]
+```
+
+## Uninstall Chart
+```
+helm uninstall device-plugin-operator
```
Comment (poussa) [inline charts/device-plugin-operator/README.md:24]: Add a comment that the CRDs are *not* uninstalled.

File: charts/device-plugin-operator/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+
```
Comment (poussa) [inline charts/device-plugin-operator/README.md:27]: Add a section which document the variables, like in the other charts.

**Review by poussa — CHANGES_REQUESTED** [review]

File: charts/device-plugin-operator/README.md (hunk 4 above)
Comment (obaranov1) [reply charts/device-plugin-operator/README.md:27]: What parameters should I define for the operator?

File: charts/device-plugin-operator/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+helm repo update
+```
+
+## Install Helm Chart
+In order to install device plugin Operator, its CRDs should be installed. 
+
+```
+helm install device-plugin-operator intel/device-plugin-operator [flags] 
+```
+You can then run helm search repo intel to see pulled charts.
```
[the remark this answers is not in the document]
Comment (obaranov1) [reply charts/device-plugin-operator/README.md]: +

**Review by mythi — APPROVED** [review]

**Review by poussa — APPROVED** [review]

**Pull request merged**